src/data/index/maintainers.yml

```diff
@@ -1,9 +1,9 @@
 maintainers:
   - name: cdai
-    friendly_name: CD&AI
+    friendly_name: Cloud Data & AI
   - name: cloud-pal
     friendly_name: Cloud PAL
   - name: watson-health
     friendly_name: Watson Health
   - name: watson-iot
```

Comment:
Should we update the directory and name too? e.g. `index/watson-iot` -> `index/ai-apps`

Reply:
Yes, I think so.